common/presenters/object-to-table-presenters.js

```diff
@@ -0,0 +1,7 @@
+const objectToTableHead = require('./object-to-table-head')
+const objectToTableRow = require('./object-to-table-row')
+
+module.exports = {
```

Comment:
This feels like this "index loader" is just to make test stubbing easier, right?

Not sure this feels quite right and doesn't follow previous patterns of an index loader being an `index.js` file within a folder. 

I'd either say use `proxyquire` to stub these presenters like we have elsewhere, and has been added in #473 or create these using the same index loader pattern with a folder.

Reply:
Fixed 👍